src/app/components/AddButton/AddButton.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import classes from './AddButton.module.css';
 import { GiLightBackpack } from 'react-icons/gi';
+import { Link } from 'react-router-dom';
 
 export default function AddButton(): JSX.Element {
   return (
     <div
       data-augmented-ui="all-hexangle-up border"
       className={classes.button_area_div}
     >
-      <button onClick={alert} className={classes.add_button_symbol}>
-        <GiLightBackpack size={42} />
-      </button>
+      <Link to="/input">
+        <button className={classes.add_button_symbol}>
+          <GiLightBackpack size={42} />
+        </button>
+      </Link>
     </div>
```

Comment:
You should try to link the whole `<div>` element, not only the icon. This way the user would have a bigger clickable area. If you put `<Link>` around the div-element, it should work.